src/components/Organisms/Header/Header.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { cleanup } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import Header from './index';
+
+describe('Test Header Component', () => {
+  afterEach(cleanup);
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<Header />, div);
+  });
+
+  it('matches snapshot', () => {
+    const tree = renderer.create(<Header />).toJSON();
```

Comment:
Snapshots can be created directly with RTL queries' return value, no need to use an extra renderer. Also, take into account that an assertion made with rendered content [provides more value than a snapshot](https://kentcdodds.com/blog/effective-snapshot-testing).

Reply:
Now using InlineSnapshot and changed the assetion